File: kotlin/src/main/kotlin/org/sammancoaching/phonebook/AlertData.kt
package org.sammancoaching.phonebook

import com.fasterxml.jackson.annotation.JsonProperty


@JvmRecord
data class AlertData(
    @field:JsonProperty("size") @param:JsonProperty("size") val size: String?,
    @field:JsonProperty("new_entry") @param:JsonProperty("new_entry") val newEntry: String?,
    @field:JsonProperty("clash") @param:JsonProperty("clash") val clash: String?
)

File: csharp/EnterprisePhonebook-TestDesign-Kata/EnterprisePhonebook/AlertData.cs
using System.Text.Json.Serialization;

namespace SammanCoaching.Phonebook;

public sealed record AlertData(
    [property: JsonPropertyName("size")] string Size,
    [property: JsonPropertyName("new_entry")] string NewEntry,
    [property: JsonPropertyName("clash")] string Clash);
